logback-core/src/main/java/ch/qos/logback/core/status/StatusManager.java

```diff
@@ -16,71 +16,75 @@
 import java.util.List;
 
 /**
- * Internal error messages (statii) are managed by instances of this interface.
- * 
+ * A component which accepts status events and notifies registered listeners.
+ * Maintains event statistics, an event buffer, and a listener registry.
+ *
  * @author Ceki G&uuml;lc&uuml;
+ * @author Mark Chesney
  */
 public interface StatusManager {
 
     /**
-     * Add a new status message.
-     * 
-     * @param status
+     * Notifies registered listeners of the specified status event and adds it to
+     * the end of the event buffer.
+     *
+     * @param status a status event
      */
     void add(Status status);
 
     /**
-     * Obtain a copy of the status list maintained by this StatusManager.
-     * 
-     * @return
+     * Returns a point-in-time snapshot of the event buffer.
+     *
+     * @return a snapshot of the event buffer
      */
     List<Status> getCopyOfStatusList();
 
     /**
-     * Return the highest level of all the statii.
-     * 
-     * @return
+     * Returns the highest level of statuses seen since instantiation.
+     *
+     * @return the highest level of statuses seen
      */
     // int getLevel();
```

Comment:
Was `StatusManager#getLevel()` commented with the intention to delete it or later uncomment it?